.circleci/config.yml

```diff
@@ -1375,3 +1406,50 @@ workflows:
       equal: [bump, << pipeline.parameters.action >>]
     jobs:
       - release-train
+
+  # to trigger tests manually, log into circleCI, select the project, a branch, and then click "Trigger Pipeline"
+  # and add a parameter called "run-manual-tests"
+  manual-test-workflow:
+    when:
+      and:
+        - not:
+            equal: [scheduled_pipeline, << pipeline.trigger_source >>]
+        - not: << pipeline.parameters.generate_snapshots >>
+        - not: << pipeline.parameters.generate_revenuecatui_snapshots >>
+        - or:
+            - matches: 
+                pattern: "^release/.*$"
+                value: << pipeline.git.branch >>
+            - equal:
+                - "manual_tests"
+                - << pipeline.trigger_source >>
+    jobs:
+      - backend-integration-tests-SK1
+      - backend-integration-tests-SK2
+      - backend-integration-tests-custom-entitlements
+      - backend-integration-tests-other
+      - build-tv-watch-and-macos
+      - build-visionos
+      - docs-build
+      - run-test-ios-14
+      - run-test-ios-15
+      - run-test-ios-16
+      - run-test-macos
+      - run-test-tvos
+      - run-test-watchos
+      - spm-custom-entitlement-computation-build
+      - spm-receipt-parser
+      - spm-release-build
+      - spm-release-build-xcode-14
+      - spm-release-build-xcode-15
+      - spm-revenuecat-ui-ios-15
+      - spm-revenuecat-ui-ios-16
+      - spm-revenuecat-ui-ios-17
+      - spm-revenuecat-ui-watchos
+      - pod-lib-lint
+      - installation-tests-cocoapods
+      - installation-tests-swift-package-manager
+      - installation-tests-custom-entitlement-computation-swift-package-manager
+      - installation-tests-carthage
+      - installation-tests-xcode-direct-integration
+      - installation-tests-receipt-parser
```

Comment:
these weren't included here but they were included in the deploy scripts, so you could have had that fail when releasing which is super annoying and waaay too late to find out that something's not working